modules/ROOT/pages/dw-core-functions-orderby.adoc

```diff
@@ -95,7 +99,7 @@ Sorts an array using the specified criteria.
 |===
 | Name   | Description
 | `array` | The list (an array) to sort.
-| `criteria` | The result of the function is used as the criteria to sort the list. It should return a simple value (String, Number, etc)
+| `criteria` | The result of the function will be used as the criteria to sort the list. It should return a simple value (String, Number, etc)
 |===
```

Comment:
are used (let's avoid future tense unless necessary to understand two events ordered in time).  Also "It should return" should be "The function returns"